sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -757,44 +757,45 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
 
   def entryPointsTask(stage: TaskKey[Attributed[File]]): Def.Initialize[Task[Seq[(String, File)]]] =
     Def.taskDyn {
+      val s = streams.value
```

Comment:
@sjrd Is this what you had in mind re: hoisting the .value calls outside the Def.task? While we're here, could we somehow grab the linkingUnit more efficiently?